symengine/sets.cpp

```diff
@@ -112,6 +112,22 @@ RCP<const Boolean> Interval::contains(const RCP<const Basic> &a) const
     return boolean(true);
 }
 
+RCP<const Set> make_set_union(const set_set &in)
```

Comment:
```suggestion
static RCP<const Set> make_set_union(const set_set &in)
```

Reply:
Thanks for the suggestion. This makes sense